From ee7ca2704974d5d7fc3b8f3158aaf129ae0e36e0 Mon Sep 17 00:00:00 2001
From: Fengguang Wu <fengguang.wu@intel.com>
Date: Tue, 6 Oct 2015 15:37:02 -0700
Subject: [PATCH 206/222] Input: ft6236 - do not explicitly set driver's owner

There is no need to explicitly set .owner for the driver, the core will do
it for us.

Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
Signed-off-by: Dmitry Torokhov <dmitry.torokhov@gmail.com>
---
 drivers/input/touchscreen/ft6236.c | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/input/touchscreen/ft6236.c
+++ b/drivers/input/touchscreen/ft6236.c
@@ -313,7 +313,6 @@ MODULE_DEVICE_TABLE(i2c, ft6236_id);
 static struct i2c_driver ft6236_driver = {
 	.driver = {
 		.name = "ft6236",
-		.owner = THIS_MODULE,
 		.of_match_table = of_match_ptr(ft6236_of_match),
 	},
 	.probe = ft6236_probe,
